src/resolutionError.ts

```diff
@@ -49,6 +50,7 @@ const HandlersByCode = {
     recordName: string;
     domain: string;
   }) => `No ${params.recordName} record found for ${params.domain}`,
+  [ResolutionErrorCode.IncorectProvider]: (params: {}) => "Provider doesn't have sendAsync or send function."
```

Comment:
```suggestion
  [ResolutionErrorCode.IncorectProvider]: (params: {}) => "Provider doesn't implement sendAsync or send method"
```
Not forcing this change, but I thought this might be a better wording